iis-wf/iis-wf-metadataextraction/src/main/java/eu/dnetlib/iis/wf/metadataextraction/AbstractMetadataExtractorMapper.java

```diff
@@ -130,13 +127,13 @@ protected void setup(Context context) throws IOException, InterruptedException {
         String excludedIdsCSV = context.getConfiguration().get("excluded.ids");
         if (excludedIdsCSV != null && !excludedIdsCSV.trim().isEmpty()
                 && !WorkflowRuntimeParameters.UNDEFINED_NONEMPTY_VALUE.equals(excludedIdsCSV)) {
-            log.warn("got excluded ids: " + excludedIdsCSV);
+            log.info("got excluded ids: " + excludedIdsCSV);
             excludedIds = new HashSet<String>(Arrays.asList(StringUtils.split(excludedIdsCSV.trim(), ',')));
         } else {
-            log.warn("got no excluded ids");
+            log.info("got no excluded ids");
         }
         // handling maximum content size
-        String maxFileSizeMBStr = context.getConfiguration().get(IMPORT_CONTENT_MAX_FILE_SIZE_MB);
+        String maxFileSizeMBStr = context.getConfiguration().get(ImportWorkflowRuntimeParameters.IMPORT_CONTENT_MAX_FILE_SIZE_MB);
         if (maxFileSizeMBStr != null && !maxFileSizeMBStr.trim().isEmpty()
                 && !WorkflowRuntimeParameters.UNDEFINED_NONEMPTY_VALUE.equals(maxFileSizeMBStr)) {
```

Comment:
We have `getParamValue`, as long as we are changing this place anyway (we could do a static import instead), maybe we can use it.